README.md

```diff
@@ -11,3 +11,32 @@ npm install --save-dev @angular-devkit/build-angular
 ```
 
 For more information on running and set-up, [visit the wiki](https://github.com/onc-healthit/lantern-front-end/wiki).
+
+## Nginx server
+
+To run with the nginx server (on a Mac):
+
+### First Time
+1. Install nginx: `brew install nginx`
+2. Copy the local nginx config file `lantern.conf` to `/usr/local/etc/nginx/servers/`
+3. Make `lantern` directory in the nginx file system:
+    - `cd /usr/local/var/www`
+    - `mkdir lantern`
+
+### Every Time
+1. Build the production version of this project: `ng build --prod`
+2. Move the `dist` directory so nginx can run it: `mv ./dist/lantern-app/ /usr/local/var/www/lantern/`
```

Comment:
I had to run `mv ./dist/lantern-app/* /usr/local/var/www/lantern/` to get this to work